wger/core/models.py

```diff
@@ -303,6 +303,12 @@ class UserProfile(models.Model):
                                     default=False)
     '''Allow anonymous read-only access'''
 
+    ''' Number of Days Reminder Weight '''
+    num_days_weight_reminder = models.IntegerField(verbose_name=_('Number of days Weight Reminder'),
+                                                   help_text=_('Num of days after the last entry to reminder the user'),
```

Comment:
This one is difficult, perhaps "Number of days after the last weight entry (enter 0 to deactivate)"?